packages/devtools_app/lib/src/debugger/program_explorer_controller.dart

```diff
@@ -130,6 +126,7 @@ class ProgramExplorerController extends DisposableController
 
   /// Marks [node] as the currently selected node, clearing the selection state
   /// of any currently selected node.
+  /// TODO(elliette): Scroll to node when selected.
```

Comment:
is this a regression? this was originally implemented by @bkonyi here https://github.com/flutter/devtools/pull/3480

Reply:
That looks like it implements scrolling within the file, this comment is for scrolling within the program explorer tree view - eg: user opens `some_file.dart` at the bottom of the tree view, and then does a hot-restart. On restart, `main.dart` is selected and opened, but the tree is still scrolled to `some_file.dart`. I've added more details to the comment to make it clearer.

Reply:
got it. On another note, after a hot restart, should the selected and opened file be some_file.dart if it is still available instead of main.dart?

Reply:
Hmm good point - added a TODO to see if that's possible